src/meta/src/manager/catalog/mod.rs

```diff
@@ -584,10 +582,7 @@ where
                             }))
                             .await
                             .into_iter()
-                            // FIXME(zehua): use `map_ok(|table| table.unwrap())` instead of this
-                            // after source node's state table has
-                            // catalog.
-                            .filter_map_ok(|table| table)
+                            .map_ok(|table| table.unwrap())
```

Comment:
Please change value of `internal_table_id_map` to non-option in `CreateMaterializedViewContext` directly, since all internal state tables have catalog now.
https://github.com/risingwavelabs/risingwave/blob/7d72451274b9fdb84914c6541dbc1d0bfa1bdf1f/src/meta/src/stream/stream_manager.rs#L65